conans/test/unittests/tools/files/checksums_test.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from conan.tools.files import check_md5, check_sha1, check_sha256
+from conans.test.utils.test_files import temp_folder
+from conans.tools import save
```

Comment:
Better not use ``from conans.tools import save`` for tests, as it will break when merged to ``develop2``. Better use ``conans.utils.files.save``